crates/wasm-mutate/src/mutators/modify_const_exprs.rs

```diff
@@ -76,7 +76,7 @@ impl<'cfg, 'wasm> Reencode for InitTranslator<'cfg, 'wasm> {
             O::RefNull { .. } | O::I32Const { value: 0 | 1 } | O::I64Const { value: 0 | 1 } => true,
             O::F32Const { value } => value.bits() == 0,
             O::F64Const { value } => value.bits() == 0,
-            O::V128Const { value } => value.i128() == 0,
+            O::Simd(SimdOperator::V128Const { value }) => value.i128() == 0,
```

Comment:
Looking at this again, what do you think of throwing all the simd ops back into `Operator`? It's already `#[non_exhaustive]` and would be nice to have a bit less churn as well in theory

Reply:
I am not yet sure if this works as intended due to the split into 2 macros and the problem that you cannot have macro invokation at enum variant definition position but I ll see how it goes. A downside to this approach is that this would result in tons of cfgs.